### DISCUSSION TITLE
Where does your Quire epub live?

### DISCUSSION THREAD

**Discussion by mmpadilla15** [thegetty/quire#641] [General]
Happy new year everyone! We are trying to figure out long-term hosting options for our current Quire epub and any potential future epubs we may produce. Our web server, unfortunately, only hosts Wordpress or Drupal sites so our epub is currently living on Netlify. We'd like to know where _your_ Quire epub is hosted. Do you host it on Github or Netlify? Or someplace else? Please let me know in the comments and, if you are so inclined, pros and cons of your choice. Thank you!

**Comment by Erin-Cecele** [comment]
Hi @mmpadilla15 the development thread #94 somewhat addresses this issue. Take a look and let me know if this is helpful or if you need more/different information. @geealbers can also speak to how Getty deploys their Quire sites.

**Reply by mmpadilla15** [reply]
@Erin-Cecele Actually, that is helpful! Thanks for pointing me to that!
